src/option.styles.ts

```diff
@@ -0,0 +1,141 @@
+import { css } from 'lit';
+
+export default [
+  css`
+    .tooltip {
+      display: block;
+    }
+
+    .container {
+      border-radius: var(--glide-core-rounding-base-radius-xs);
+      box-sizing: border-box;
+      color: var(--glide-core-color-static-text-default);
+      cursor: pointer;
+      font-family: var(--glide-core-typography-family-primary);
+      font-size: var(--glide-core-typography-size-body-default);
+      font-weight: var(--glide-core-typography-weight-regular);
+      inline-size: 100%;
+      max-inline-size: 21.875rem;
+      padding-block: var(--glide-core-spacing-base-xxs);
+      padding-inline: var(--glide-core-spacing-base-sm);
+      transition: background-color 100ms ease-in-out;
```

Comment:
Nit. Could swap this to a variable:


```suggestion
      transition: background-color var(--glide-core-duration-fast-02) ease-in-out;
```

Not sure what timing function they'd want, but could swap that one out too.